orchagent/intfsorch.h

```diff
@@ -35,7 +36,7 @@ class IntfsOrch : public Orch
     sai_object_id_t getRouterIntfsId(const string&);
     bool isPrefixSubnet(const IpPrefix&, const string&);
     string getRouterIntfsAlias(const IpAddress &ip, const string &vrf_name = "");
-
+    string getRifRateFlexCounterTableKey(string key);
```

Comment:
>getRifRateFlexCounterTableKey [](start = 11, length = 29)

above line uses RouterIntfs, here uses Rif. Are they different?

Reply:
The latter refers to RIF_RATE Flex Counter group instead of Router Interface.